{
    "type": "vulnerability",
    "spec_version": "2.1",
    "id": "vulnerability--4c40715c-9c54-4c64-a204-4872fbac6812",
    "created": "2024-08-13T16:04:09.809236Z",
    "modified": "2024-08-13T16:04:09.809241Z",
    "name": "No Title Available",
    "description": "libautotrace.a in AutoTrace 0.31.1 allows remote attackers to cause a denial of service (invalid read and SEGV), related to the input_pnm_reader function in input-pnm.c:243:3.",
    "external_references": [
        {
            "source_name": "cve",
            "external_id": "CVE-2017-9155"
        }
    ],
    "references": [
        {
            "tags": [
                "x_refsource_MISC"
            ],
            "url": "https://blogs.gentoo.org/ago/2017/05/20/autotrace-multiple-vulnerabilities-the-autotrace-nightmare/"
        }
    ]
}